public/lessons/react/jsx_and_the_virtual_dom.md

```diff
@@ -36,111 +37,90 @@ ReactDOM.render(React.createElement(
 </div>
 ```
 
-# What we learned
+# Key Concepts
 
 - JSX feels like HTML (inside of JavaScript)
 - JSX compiles to ReactDOM code
 - JSX is a 'syntax extension' to JavaScript
+- React can only render a single JSX component
+  - **but** that component can have as many children as you want
 - ReactDOM builds the HTML from the virtual DOM
 
 # JSX can embed JavaScript expressions
 
-Building a greeter
+- When writing JSX you can escape back to JavaScript using curly braces `{}`
```

Comment:
This is a good point to drive home, aside from React in general, students have seemed to have some confusion over what 'escaping to javascript' means at first